x-pack/plugins/triggers_actions_ui/public/application/sections/action_connector_form/action_type_form.tsx

```diff
@@ -161,18 +165,21 @@ export const ActionTypeForm = ({
   const [availableActionVariables, setAvailableActionVariables] = useState<ActionVariable[]>([]);
   const defaultActionGroup = actionGroups?.find(({ id }) => id === defaultActionGroupId);
   const selectedActionGroup =
-    actionGroups?.find(({ id }) => id === actionItem.group) ?? defaultActionGroup;
+    (!isSystemAction && actionGroups?.find(({ id }) => id === actionItem.group)) ||
+    defaultActionGroup;
   const [actionGroup, setActionGroup] = useState<string>();
   const [actionParamsErrors, setActionParamsErrors] = useState<{ errors: IErrorObject }>({
     errors: {},
   });
   const [actionThrottle, setActionThrottle] = useState<number | null>(
-    actionItem.frequency?.throttle
+    !isSystemAction && actionItem.frequency?.throttle
       ? getDurationNumberInItsUnit(actionItem.frequency.throttle)
       : null
   );
   const [actionThrottleUnit, setActionThrottleUnit] = useState<string>(
-    actionItem.frequency?.throttle ? getDurationUnitValue(actionItem.frequency?.throttle) : 'h'
+    !isSystemAction && actionItem.frequency?.throttle
+      ? getDurationUnitValue(actionItem.frequency?.throttle)
+      : 'h'
```

Comment:
Will this default value affects the way system actions are being executed?

Reply:
We created a new component for system actions. Any functionality around throttling is not used.